wagtail/images/templatetags/wagtailimages_tags.py

```diff
@@ -6,7 +6,8 @@
 from django.urls import NoReverseMatch
```

Comment:
```suggestion
from django.core.exceptions import ImproperlyConfigured
from django.template.base import FilterExpression
from django.template.loader import render_to_string
from django.urls import NoReverseMatch
```

and there's also a missing import:

```py
from django.conf import settings
```